### PR TITLE
Update Welcome section with target audience and instructional design

### PR DESCRIPTION
**Pull request by lwjohnst86**

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -35,33 +30,27 @@ lof: yes
...
-
+Computers are as essential for science and research as telescopes,
+test tubes, and libraries. Researchers in many fields are increasingly
+needing to know how to create, use, and share software given modern
+demands and requirements for research. However, most introductions to
+programming focus on developing commercial applications and not
+typically on the type of tasks and activities that researchers more
+commonly do, like exploring problems and answering questions. This
+book will show you how to create software as part of your research
+cycle and work where you either work alone or as part of a team. For
```
Comment (mbonsma) [inline index.Rmd:41]:
```suggestion
cycle whether you work alone or as part of a team. For
```

File: index.Rmd [hunk 2]
```diff
@@ -35,33 +30,27 @@ lof: yes
...
-For researchers and data scientists who can write functions to create programs that are several pages long,
-and who want to be more productive and have more confidence in their results,
-this book provides a pragmatic, tools-based introduction to research software engineering.
-Unlike material courses aimed at computer scientists and professional software developers,
-this book uses data analysis as a motivating example
-and assumes that the learner's ultimate goal is to answer questions rather than ship products.
-
+Computers are as essential for science and research as telescopes,
+test tubes, and libraries. Researchers in many fields are increasingly
+needing to know how to create, use, and share software given modern
```
Comment (mbonsma) [inline index.Rmd:35]:
```suggestion
test tubes, and libraries. Researchers in many fields need
to know how to create, use, and share software given modern
```

File: index.Rmd (hunk 1 above)
Comment (mbonsma) [inline index.Rmd:41]:
```suggestion
cycle whether you work alone or as part of a team. For
```

File: index.Rmd [hunk 3]
```diff
@@ -35,33 +30,27 @@ lof: yes
...
+demands and requirements for research. However, most introductions to
+programming focus on developing commercial applications and not
+typically on the type of tasks and activities that researchers more
+commonly do, like exploring problems and answering questions. This
+book will show you how to create software as part of your research
+cycle and work where you either work alone or as part of a team. For
+researchers who spend most of their time building software packages,
```
Comment (mbonsma) [inline index.Rmd:42]:
```suggestion
researchers who build software packages,
```

Even if it's not most of your time!

File: index.Rmd [hunk 4]
```diff
@@ -35,33 +30,27 @@ lof: yes
...
+engineer][rse].
+
+For researchers and data scientists who can write functions to create
+programs that are hundreds of lines long and who want to be more
+productive and have more confidence in their results, this book
+provides a pragmatic, tools-based introduction to research software
+engineering. Unlike courses aimed at computer scientists and
+professional software developers, this book uses data analysis as the
+motivating example and assumes that the learner's ultimate goal is to
+answer questions rather than ship products.
```
Comment (mbonsma) [inline index.Rmd:53]: Love this!

File: index.Rmd [hunk 5]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
 
+## Target audience
+
+While the learning content of this book targets the specific learning personas
+described in FIXME, we wrote this book with three audiences in mind:
+
+1. **Instructors** who would use this book as the basis for a course or workshop
+of their own. Throughout the book there are sections dedicated to this reader
+that give information or instructions on how to use or modify the material for
+the instructors specific context.
```
Comment (mbonsma) [inline index.Rmd:76]:
```suggestion
the instructor's specific context.
```

File: index.Rmd [hunk 6]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+2. **Learners in a class setting** who would be taking this course in a more
+formal class or workshop setting that is lead by an instructor(s). Much of the
```
Comment (mbonsma) [inline index.Rmd:78]:
```suggestion
formal class or workshop setting that is lead by an instructor. Much of the
```

I don't know, we can keep the (s) but it just feels kind of awkward and maybe not necessary?
Alternate suggestion: 
```suggestion
formal class or workshop setting that is lead by an instructor or instructors. Much of the
```

File: index.Rmd [hunk 7]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+and describe where necessary.
+
+## Instructional design
+
+The instructional design of the learning material contained in this
+book revolves around three key concepts:
+
+1. *Participatory live-coding* lessons, where participants join with
+instructors to write and troubleshoot code step-by-step. We believe
+that this encourages participants to actively engage with the
```
Comment (mbonsma) [inline index.Rmd:92]:
```suggestion
that this method encourages participants to actively engage with the
```

File: index.Rmd [hunk 8]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+material, to build muscle memory through typing, and to learn how to
+handle mistakes, rather than passively observing content.
+2. *In-class and out-of-class exercises* that includes independent
```
Comment (mbonsma) [inline index.Rmd:95]:
```suggestion
2. *In-class and out-of-class exercises* that include independent
```

File: index.Rmd [hunk 9]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+reading of assigned material and hands-on, practical work.
+The hands-on exercises are interspersed throughout the live-coding
+sessions to complement and reinforce the content being learned as well
```
Comment (mbonsma) [inline index.Rmd:98]:
```suggestion
sessions to complement and reinforce the content as well
```

File: index.Rmd [hunk 10]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+as give learners the opportunity to work through the code and problems
+at their own pace. Reading activities are used to build concept-heavy
+knowledge.
+3. *Project- and problem-based assignments* that includes both independent
```
Comment (mbonsma) [inline index.Rmd:102]:
```suggestion
3. *Project- and problem-based assignments* that include both independent
```

File: index.Rmd [hunk 11]
```diff
@@ -76,6 +65,47 @@ The original source material for this course is found on the
...
+and group-based work. For the project assignments, learners reinforce
+what they learned through a new problem and challenge. The independent
+work prepares them for the group-based (though optional) work, where
```
Comment (mbonsma) [inline index.Rmd:105]:
```suggestion
work prepares them for the optional group-based work, where
```

**Review by mbonsma — APPROVED** [review]
Looks awesome! I made some very minor suggestions for conciseness.

**Comment by lwjohnst86** [comment]
Thanks @mbonsma!

@joelostblom and @k8hertweck for #59, instead of editing `index.Rmd` in your own branches can you comment instead on this PR and the text of index in this branch?

**Comment by lwjohnst86** [comment]
Merging this and moving over into preface as per our discussion.

**Pull request merged**